components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationResourceManager.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2023, WSO2 LCC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We don't need to change the year for modified files.
Just chaning Inc. --> to LLC. would do